src/Replacers/CsrfTokenReplacer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Spatie\ResponseCache\Replacers;
+
+use Symfony\Component\HttpFoundation\Response;
+
+class CsrfTokenReplacer implements Replacer
+{
+    public function replacedBy(): string
```

Comment:
Since we do not call this from the outside, it might be better make this protected. We could also just use a protected property.